src/wallet/index.ts

```diff
@@ -190,4 +200,111 @@ class Wallet {
   }
 }
 
+function signWithKeypair(
```

Comment:
This can be a private method on the `Wallet` class - then you don't need to pass in the public and private keys